cmd/query/app/static_handler.go

```diff
@@ -74,9 +88,20 @@ func NewStaticAssetsHandler(staticAssetsRoot string, uiConfig string) (*StaticAs
 		bytes, _ := json.Marshal(config)
 		configString = fmt.Sprintf("JAEGER_CONFIG = %v", string(bytes))
 	}
+	indexBytes = configPattern.ReplaceAll(indexBytes, []byte(configString+";"))
+	if options.BasePath == "" {
+		options.BasePath = "/"
+	}
+	if options.BasePath != "/" {
+		if !strings.HasPrefix(options.BasePath, "/") || strings.HasSuffix(options.BasePath, "/") {
```

Comment:
Is it possible to add a log statement notifying that what people see in the HTML is *not* what will be served to clients?

Reply:
could you elaborate why you think this would be useful? We're modifying the internal file of the app, not something that users provide.

Reply:
Mainly because it's deviating from what is "expected": if I had a problem with the static resources (like the original problem this PR is solving) and were to check the HTML, I would wonder why it's not the same as what my browser is seeing. A log entry at `debug` level would suffice, IMO.

Reply:
well, still not convinced why someone needs to know what happens internally, e.g. we also override the config, and in both cases we only do that when requested via cli arguments.  

Let's do it separately, because the constructor doesn't take a logger and the unit tests  explicitly verify that nothing is logged, so it's not as trivial a change as it sounds for a feature that seems rather marginal.